LEEME.md

```diff
@@ -1,13 +1,13 @@
 <!--multilang v0 es:LEEME.md en:README.md -->
 
-
 <!--lang:es-->
 <!--lang:en--]
 [!--lang:*-->
 # backend-plus
 
 <!--lang:es-->
 Backend for typed-controls
+Backend-plus es un framework generador de ABMs escrito en Node.
```

Comment:
No es un framework generador en el sentido de que no genera código ni otro tipo de archivos. En todo caso es más parecido a un intérprete que a un navegador. Y tampoco se centra solo en ABMs. Cierto es que la primera oración es menos cierta "Backend for typed-controls"